edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -195,7 +195,7 @@ <h2 id="acessible-error-title">{% trans "Cannot Start Proctored Exam" %}</h2>
       },
       success: function (data) {
         newWindow.location = "{{software_download_url|safe}}";
-        setTimeout(poll_until_ready, 15000);
+        // setTimeout(poll_until_ready, 15000);
```

Comment:
probably better to delete this altogether. Why is this necessary?

Reply:
not sure if we're going to want to revive it. but I'll remove it for now.

Reply:
I'd remove the dead code as well; it's better habit to be in to not leave this stuff in place, since it inflates what needs to be sent to the browser.